### PR TITLE
Add GATEWAY_URL value

### PR DESCRIPTION
**Pull request by Gogetter**
### Add Gateway url configuration to cnp-flux-config repo ###

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/common/family-public-law/fpl-case-service.yaml [hunk 1]
```diff
@@ -34,6 +34,7 @@ spec:
         DOCMOSIS_TORNADO_URL: https://docmosis-development.platform.hmcts.net
         IDAM_CLIENT_ID: fpl_case_service
         IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-*.service.core-compute-aat.internal/oauth2/callback
+        GATEWAY_URL: http://gateway-ccd.aat.platform.hmcts.net
```
Comment (geek-soft) [inline k8s/aat/common/family-public-law/fpl-case-service.yaml]: Please do same for prod otherwise POD won't start in PROD.

Comment (geek-soft) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml]: And it should be `https://gateway-ccd.aat.platform.hmcts.net`

Comment (Gogetter) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml]: `https` for both aat and prod?

Comment (geek-soft) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml]: yes, always favour https over http unless https is not supported

Comment (Gogetter) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml]: updated

File: k8s/prod/common/family-public-law/fpl-case-service.yaml [hunk 2]
```diff
@@ -34,6 +34,7 @@ spec:
         DOCMOSIS_TORNADO_URL: https://docmosis.platform.hmcts.net
         IDAM_CLIENT_ID: fpl_case_service
         IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-prod.service.core-compute-prod.internal/oauth2/callback
+        GATEWAY_URL: http://gateway-ccd.prod.platform.hmcts.net
```
Comment (geek-soft) [inline k8s/prod/common/family-public-law/fpl-case-service.yaml]: This is wrong. Where did you take it from. It should be `https://gateway.ccd.platform.hmcts.net`.

Comment (Gogetter) [reply k8s/prod/common/family-public-law/fpl-case-service.yaml]: thanks for spotting this. Was in `prod.tfvars`. Now updated

**Pull request merged**